RecoEgamma/EgammaElectronProducers/src/LowPtGsfElectronSeedHeavyObjectCache.cc

```diff
@@ -0,0 +1,170 @@
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/ParticleFlowReco/interface/PreId.h"
+#include "DataFormats/ParticleFlowReco/interface/PFCluster.h"
+#include "DataFormats/ParticleFlowReco/interface/PFClusterFwd.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "RecoEgamma/EgammaElectronProducers/interface/LowPtGsfElectronSeedHeavyObjectCache.h"
+#include "CommonTools/MVAUtils/interface/GBRForestTools.h"
+#include "TMVA/MethodBDT.h"
+#include "TMVA/Reader.h"
+#include <string>
+
+namespace lowptgsfeleseed {
+
+  ////////////////////////////////////////////////////////////////////////////////
+  //
+  std::vector<float> Features::get() {
+    std::vector<float> output = { 
+      trk_pt_,
+      trk_eta_,
+      trk_phi_,
+      trk_p_,
+      trk_nhits_,
+      trk_high_quality_,
+      trk_chi2red_,
+      rho_,
+      ktf_ecal_cluster_e_,
+      ktf_ecal_cluster_deta_,
+      ktf_ecal_cluster_dphi_,
+      ktf_ecal_cluster_e3x3_,
+      ktf_ecal_cluster_e5x5_,
+      ktf_ecal_cluster_covEtaEta_,
+      ktf_ecal_cluster_covEtaPhi_,
+      ktf_ecal_cluster_covPhiPhi_,
+      ktf_ecal_cluster_r9_,
+      ktf_ecal_cluster_circularity_,
+      ktf_hcal_cluster_e_,
+      ktf_hcal_cluster_deta_,
+      ktf_hcal_cluster_dphi_,
+      preid_gsf_dpt_,
+      preid_trk_gsf_chiratio_,
+      preid_gsf_chi2red_,
+      trk_dxy_sig_
```

Comment:
@bainbrid you are always passing the displacement to the model, even if the model does not use it. I guess this is fine since is the last element of an array, but if it wasn't it would create serious problems